google/reader.go

```diff
@@ -107,3 +132,69 @@ func (r *GCPReader) ListNetworks(ctx context.Context, filter string) ([]compute.
 	}
 	return networks, nil
 }
+
+// ListHealthCheck returns a list of health checks in the project
+func (r *GCPReader) ListHealthCheck(ctx context.Context, filter string) ([]compute.HealthCheck, error) {
+	ns := compute.NewHealthChecksService(r.compute)
+	// 500 is the current `maxResults`
```

Comment:
Maybe this value should be configurable in `NewGcpReader` ? If not, at least some const

Reply:
Yes defining a constant for this, if it's the same on all the resources, would be good.